sdk/identity/identity/src/credentials/authorizationCodeCredential.ts

```diff
@@ -38,7 +38,7 @@ export class AuthorizationCodeCredential implements TokenCredential {
    * the authorization code flow to obtain an authorization code to be used
    * with this credential.  A full example of this flow is provided here:
    *
-   * https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/identity/identity/samples/authorizationCodeSample.ts
+   * https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/identity/identity/samples/manual/authorizationCodeSample.ts
```

Comment:
Important to note that these samples in the `manual` folder _will not_ be published to docs.microsoft.com. They will only be a kind of "developer reference" that lives in the repo.

Reply:
That is ok by me!